cmd/src/users.go

```diff
@@ -97,4 +97,5 @@ type SiteUser struct {
 	Email        string
 	SiteAdmin    bool
 	LastActiveAt string
+	DeletedAt    string
```

Comment:
Why `string` and not `time.Time`?

Reply:
Its just a little easier to check for the null values of a string here and the graphQL api defines this as a string 🤔 , and in this case I'm only using this field to check if the field is returning non null
<img width="306" alt="Screen Shot 2023-01-12 at 11 55 26 PM" src="https://user-images.githubusercontent.com/13024338/212267693-ab6308cf-a813-4e0d-9363-19c034761ca8.png">

Reply:
Got it, makes sense